code-challenges.test.js

```diff
@@ -14,33 +14,122 @@
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
+describe("sortRemain",() => {
+  const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+  // Expected output: [ 2, 0, -1, 0 ]
+  const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+  // Expected output: [ 2, 1, -1 ]    
+  it("filter the integers from an array, and retrive remainder after dividing by three", () => {
+    expect(sortRemain(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    expect(sortRemain(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+} )
+
+// yarn run v1.22.19
+// warning package.json: No license field
+// $ /Users/learnacademy/Desktop/week-6-assessment-salas89jl/node_modules/.bin/jest
+//  FAIL  ./code-challenges.test.js
+//     ✕ filter the integers from an array, and retrive remainder after dividing by three
+
+//   ● sortRemain › filter the integers from an array, and retrive remainder after dividing by three
+
+//     ReferenceError: sortRemain is not defined
 
 // b) Create the function that makes the test pass.
 
+// Pseudo Code:
+// 1. Create a function named sortRemain
+// 2. Input will be an array of mixed media
+// 3. copy filtered array into new a variable called numArray where values that are only integers.
+// 4. Once integers have been filtered use .map to access all numbers and modulo by 3.
+// 5. Return the numArrray.
+
+sortRemain = (arr) => {
+ numArray = arr.filter(num => Number.isInteger(num)).map(num => num % 3)
+  return numArray
+}
+
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+describe("sumCube", () => {
+  const cubeAndSum1 = [2, 3, 4]
+  // Expected output: 99
+  const cubeAndSum2 = [0, 5, 10]
+  // Expected output: 1125
+  it("takes in numsArray and sums up all numbers and then performs a cube operation", () => {
+    expect(sumCube(cubeAndSum1)).toEqual(99)
+    expect(sumCube(cubeAndSum2)).toEqual(1125)
+  })
+})
+
+// yarn run v1.22.19
+// warning package.json: No license field
+// $ /Users/learnacademy/Desktop/week-6-assessment-salas89jl/node_modules/.bin/jest
+//  FAIL  ./code-challenges.test.js
+//   sumCube
+//     ✕ takes in numsArray and sums up all numbers and then performs a cube operation
 
 // b) Create the function that makes the test pass.
+
+// Pseudo Code:
+// 1. Create a function named sumCube.
+// 2. Input: function will take an array of numbers 
+// 3. function will need to access all of the numbers in the array and store them in a variable total
+// 4. Use .map to cycle through each value and do a 1expo3 or num * num * num
+// 5. Use .reduce to access and add up all the modified values in the new array. 
+// 6. Return the total
```

Comment:
Pseudo code is confusing here; can't tell if you understand the logic or not. I would work on explaining what is going on in your code in a clearer way